src/battle-scene.ts

```diff
@@ -2506,32 +2507,46 @@ export default class BattleScene extends SceneBase {
    * @returns
    */
   getMysteryEncounter(override: MysteryEncounter): MysteryEncounter {
+    const biomeMysteryEncounters = mysteryEncountersByBiome.get(this.arena.biomeType);
     // Loading override or session encounter
     let encounter: MysteryEncounter;
-    if (!Utils.isNullOrUndefined(Overrides.MYSTERY_ENCOUNTER_OVERRIDE) && Overrides.MYSTERY_ENCOUNTER_OVERRIDE < allMysteryEncounters.length) {
-      encounter = allMysteryEncounters[Overrides.MYSTERY_ENCOUNTER_OVERRIDE];
+    if (!Utils.isNullOrUndefined(Overrides.MYSTERY_ENCOUNTER_OVERRIDE) && Overrides.MYSTERY_ENCOUNTER_OVERRIDE < biomeMysteryEncounters.length) {
```

Comment:
Override should be able to force any encounter, not just one that can appear for that biome.

Reply:
Good catch